### PR TITLE
[12.0][MIG] ao_stock_account

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
Blocked by a dependency: stock_account_valuation_report

### CODE REVIEW

**Comment by HviorForgeFlow** [comment]
Error detected on python 3.5.2
```
    fromisoformat = dt.fromisoformat
AttributeError: type object 'datetime.datetime' has no attribute 'fromisoformat'
```

File: ao_stock_account/README.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+* When reversing a transfer, set the field 'to refund' to true by
+  default (#20925)
+
+Installation
+============
+
+If you use a **python version lower than 3.7**, you will have an extra
+dependency. Install it by running::
+
+    pip install backports-datetime-fromisoformat
```
Comment (LoisRForgeFlow) [inline ao_stock_account/README.rst:29]: see this @hveficent

Comment (HviorForgeFlow) [reply ao_stock_account/README.rst:29]: I saw it after I had fixed it :smile:

**Comment by HviorForgeFlow** [comment]
@lreficent dependency is already merged in OCA/stock-logistics-reporting

**Pull request merged**